.github/workflows/build-rpms.yml

```diff
@@ -0,0 +1,29 @@
+name: Build RPMs
```

Comment:
In addition to updating the RPM spec, this is pulling in a lot of extra files I'm not sure we need.  Can we change it so that it just captures the spec update?

Reply:
Rojo added this file